src/ui/users/list/UserListMenuActions.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { DropdownKebab, MenuItem } from 'patternfly-react';
+
+class UserListMenuActions extends Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(handler) {
+    return (ev) => {
+      ev.preventDefault();
+      if (handler) { handler(this.props); }
+    };
+  }
+
+  render() {
+    const { onClickDelete } = this.props;
+    const editLabel = (
+      <FormattedMessage id="app.edit" values={{ username: this.props.username }} />
+    );
+    const manageAuthLabel = (
+      <FormattedMessage id="user.action.manageAuth" values={{ username: this.props.username }} />
+    );
+    const editProfileLabel = (
+      <FormattedMessage id="user.action.editProfile" values={{ username: this.props.username }} />
+    );
+    const viewProfileLabel = (
+      <FormattedMessage id="user.action.viewProfile" values={{ username: this.props.username }} />
+    );
+    return (
+      <DropdownKebab pullRight id={`${this.props.username}-actions`}>
+
+        <MenuItem
+          className="UserListMenuAction__menu-item-edit"
+          onClick={this.handleClick()}
```

Comment:
Please remove there empty handlers and teh handleClick method: preventDefault should be done automatically by MenuItem component